qiskit_experiments/randomized_benchmarking/rb_utils.py

```diff
@@ -0,0 +1,260 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019-2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+RB Helper functions
+"""
+
+from typing import Tuple, Dict, Optional, Iterable, List
+import numpy as np
+from qiskit import QiskitError, QuantumCircuit
+from qiskit.providers.backend import Backend
+
+
+class RBUtils:
+    """A collection of utility functions for computing additional data
+    from randomized benchmarking experiments"""
+
+    @staticmethod
```

Comment:
Perhaps you want to decorate with `@classmethod`? Same for others. (i.e. there is no code creating an instance of `RBUtil`)

Reply:
This is what `@staticmethod` does. In `@classmethod` the class itself is a parameter (for setting class variables etc.) and I don't need it here.